### PR TITLE
move setting to background job

### PR DESCRIPTION
**Pull request by reedstrm**

### CODE REVIEW

File: db/background_migrate/20200617142820_cache_points_for_existing_tasks.rb [hunk 1]
```diff
@@ -60,9 +62,11 @@ def up
     change_column_null :tasks_tasks, :published_points_after_due, false
     change_column_null :tasks_tasks, :is_provisional_score_before_due, false
     change_column_null :tasks_tasks, :is_provisional_score_after_due, false
+    change_column_null :tasks_tasks, :gradable_steps_count, false
```
Comment (Dantemss) [inline db/background_migrate/20200617142820_cache_points_for_existing_tasks.rb:65]: gradable_step_count (and below too)

File: app/subsystems/tasks/models/task.rb [hunk 2]
```diff
@@ -672,4 +672,8 @@ def closes_at_on_or_after_due_at
     errors.add(:closes_at, 'must be on or after due_at')
     throw :abort
   end
+
+  def gradable_step_count
+      super.nil? ? 0 : super
+  end
```
Comment (Dantemss) [inline app/subsystems/tasks/models/task.rb:678]: This method needs to be public (move it up in the file).

File: db/background_migrate/20200617142820_cache_points_for_existing_tasks.rb (hunk 1 above)
Comment (reedstrm) [reply db/background_migrate/20200617142820_cache_points_for_existing_tasks.rb:65]: Gaaaa!

Comment (reedstrm) [reply db/background_migrate/20200617142820_cache_points_for_existing_tasks.rb:65]: at least I was consistent (in that file)

File: app/subsystems/tasks/models/task.rb (hunk 2 above)
Comment (reedstrm) [reply app/subsystems/tasks/models/task.rb:678]: Ah found it "protected"

**Pull request merged**